test/models/language_model_test.rb

```diff
@@ -81,6 +81,72 @@ class LanguageModelTest < ActiveSupport::TestCase
     assert list.include?("alpaca:medium")
   end
 
+  # Functionality in LanguageModel::Export concern
```

Comment:
These tests look good, but I've been putting tests for a concern into the corresponding test file so that would be: `test/models/language_model/export_test.rb`. Then you don't need this comment